pages/dug_12/4-shameless-plug.md

```diff
@@ -0,0 +1,619 @@
+---
+theme: @fs/repo/slidev-template/theme
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+## &#x1F44B; DUG#12 Shameless Plug &#x1F44B;
+
+<center><img src="/@fs/repo/img/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px"></center>
+
+---
+
+<br>
+
+<center><img src="/@fs/repo/img/dug_11/shameless_plug.png" width="340"></center>
+
+<br>
+
+- 🌱 **Growth & Transparency:** Showcasing our evolution and commitment to open-source.
+- 📚 **Historical Record:** A resource for 3mdeb, future customers, and the privacy/security community.
+- 📈 **Explore Business Model:** Learn from our open-source firmware journey, including potential pitfalls.
+
+---
+
+## What we will talk about
+
+<br>
+
+- Hardware,
+- Services,
+- Pace Enterprise Training,
+- Dasharo Pro/Enterprise Package
+- Accessories
+- Everything available in 3mdeb shop: https://shop.3mdeb.com
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Hardware
+
+---
+
+<center><img src="/@fs/repo/img/dug_9/shop_sdwire.png" width="700"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/shop/open-source-hardware/sdwire/](https://shop.3mdeb.com/shop/open-source-hardware/sdwire/)
+</center>
+
+<!--
+
+We are in the middle of restock, new batch available half of December 2025.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_9/shop_twonkie.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/shop/open-source-hardware/twonkie-usb-c-sniffer/](https://shop.3mdeb.com/shop/open-source-hardware/twonkie-usb-c-sniffer/)
+</center>
+
+<!--
+
+Around 30 remain in stock.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_usbsniffer.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/](https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/)
+</center>
+
+<!--
+
+Awaiting final production offer.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_usbarmory.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/usb-armory-mk-ii/](https://shop.3mdeb.com/product/usb-armory-mk-ii/)
+</center>
+
+<!--
+
+Q1 2026
+
+-->
+
+---
+
+<center>
+
+### PCI USB controller that enables device certification for Qubes OS
+</center>
+<!--
+
+Q1 2026 ?
+
+-->
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Dasharo Supported Hardware
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_asrock.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/](https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/)
+</center>
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_gigabyte-mz33.png" width="500"></center>
+
```

Comment:
@Stanislaw-bnk 
There is some information missing here – a comment for the presenter stating that 
1. The product will be released to the shop in Q1'26
2. Initially, it will be a version with one set of components – exactly the ones we tested, because we take responsibility for their compatibility.
3. Over time, the product will be developed, as discussed with AsRock.

Reply:
e7778a54ada501079820e18feeb7e01a9f637a56